src/libraries/System.Reflection.Emit.ILGeneration/tests/ILGenerator/Emit5Tests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public sealed class ILGeneratorEmit5
+    {
+        public interface IHelper
+        {
+            int PassThrough(in int value);
+        }
+
+        public sealed class Helper : IHelper
+        {
+            public int PassThrough(in int value) => value;
+        }
+
+        [Fact]
+        public void TestEmitCallFunctionWithInArgumentFromDynamicMethod()
+        {
+            var methodToCall = typeof(IHelper).GetMethod("PassThrough");
```

Comment:
nit: avoid `var` when type is not obvious (i.e. use `MethodInfo methodToCall =`). Here and below in a couple of places.